dpnp/linalg/dpnp_utils_linalg.py

```diff
@@ -2681,24 +2709,22 @@ def dpnp_svd(
     )
     s_h = dpnp.empty_like(a_h, shape=(k,), dtype=s_type)
 
-    ht_lapack_ev, _ = li._gesvd(
+    ht_ev, gesvd_ev = li._gesvd(
         exec_q,
         jobu,
         jobvt,
         a_h.get_array(),
         s_h.get_array(),
         u_h.get_array(),
         vt_h.get_array(),
-        [a_copy_ev],
+        depends=[copy_ev],
     )
+    _manager.add_event_pair(ht_ev, gesvd_ev)
 
     if batch_call:
```

Comment:
This condition now makes no sense in the new logic using `SequentialOrderManager` and looks the same with 2725 line.
I think you can remove this and `batch_call=False` parameter in `dpnp_svd`